packages/uui-input/lib/uui-input.element.ts

```diff
@@ -123,10 +123,10 @@ export class UUIInputElement extends UUIFormControlMixin(
    * @default
    */
   @property({ attribute: 'maxlength-message' })
-  maxlengthMessage: string | ((max: number, current: number) => string) = (
```

Comment:
Changing this from `current` to `exceeded` is a breaking change.
We must remember we are not the only ones using the UI Library. So there may be someone who already uses this and parses in those values. So we need to have the logiv locally. 
I would suggest to resolve in this way: `Maximum ${max} characters, ${current-max} too many`